lib/danger/scm_source/git_repo.rb

```diff
@@ -19,6 +19,34 @@ def diff_for_folder(folder, from: "master", to: "HEAD")
       self.log = repo.log.between(from, to)
     end
 
+    def renamed_files
+      # Get raw diff with --find-renames --diff-filter
+      diff = exec(
+        "diff #{self.diff.from} #{self.diff.to} --find-renames --diff-filter=R"
```

Comment:
No, it's [default](https://git-scm.com/docs/git-diff) git flags. 

Unfortunately we can't use it cause of `--find-renames` flag. 
We need to pass it to diff query cause older versions of git don't have this option as default (That caused errors on travis ci which has git version 1.8.0).